typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumBscLumiaLUMIAWarpConfig.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  ChainMap,
+  RouterConfig,
+  TokenRouterConfig,
+  TokenType,
+} from '@hyperlane-xyz/sdk';
+import { objMap } from '@hyperlane-xyz/utils';
+
+// Lumia Team
+const owner = '0x8bBA07Ddc72455b55530C17e6f6223EF6E156863';
+
+export const getEthereumBscLUMIAWarpConfig = async (
+  routerConfig: ChainMap<RouterConfig>,
+): Promise<ChainMap<TokenRouterConfig>> => {
+  const ethereum = {
+    type: TokenType.collateral,
+    token: '0xD9343a049D5DBd89CD19DC6BcA8c48fB3a0a42a7',
+  };
+
+  const bsc = {
+    type: TokenType.synthetic,
+  };
+
+  const lumia = {
+    type: TokenType.native,
+  };
+
+  const configMap = {
+    ethereum,
+    bsc,
+    lumia,
+  };
+
+  const merged = objMap(configMap, (chain, config) => ({
+    ...routerConfig[chain],
+    ...config,
+    owner,
+    ownerOverrides: {
+      proxyAdmin: owner,
```

Comment:
nit: this does not need to be explicitly defined and the check will fall back to using owner if it is not defined. I'm not too opinionated on whether you think it's better to be explicit from readability perspective